src/app/signup/page.tsx

```
@@ -0,0 +1,287 @@
+            <CardTitle className="text-xl">Sign Up</CardTitle>
+            <CardDescription>Create your account to get started with Alimana</CardDescription>
+            <Button
+              variant="outline"
+              onClick={handleGoogleSignup}
+              className="w-full bg-transparent"
+            >
+              <svg
+                className="mr-2 h-4 w-4"
+                viewBox="0 0 48 48"
+                // xmlns="http://www.w3.org/2000/svg"
+              >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)